docs/apb_cli.md

```diff
@@ -160,6 +160,12 @@ If you are using a namespace other than the default `openshift` namespace to hos
 apb push -o --namespace <namespace>
 ```
 
+This is assuming that the user has the cluster-admin role assigned to the user that is logged in. i.e.:
+```
```

Comment:
do we want to include an OR `oc login -u system:admin`

Reply:
Can't use system:admin when we require a token unfortunately.